lib/httpErrors.js

```diff
@@ -11,6 +11,7 @@ Object.keys(statusCodesMap).forEach(code => {
 function normalize (code, msg) {
   if (code === '414') return 'uriTooLong'
   if (code === '505') return 'httpVersionNotSupported'
+  if (code === '418') return 'imateapot'
```

Comment:
we should not do this.

Reply:
This looks like the cleanest way since it will generate `imateapot` or `imaTeapot` based on node versions and breaks entire test suite 😄 

See https://github.com/fastify/fastify-sensible/blob/1961477c14f9232747d93aa841d790d57bd30fb0/lib/httpErrors.js#L96